api/src/test/resources/registration_app.json

```diff
@@ -140,35 +140,6 @@
               ]
             }
           ]
-        },
-        {
-          "id": "ethnicity",
-          "label": "Ethnicity",
-          "questions": [
-            {
-              "legend": "acme.app.registerPatient.section.ethnicity",
-              "fields": [
-                {
-                  "type": "obs",
-                  "formFieldName": "obs.ACME:912",
-                  "label": "acme.app.registerPatient.ethnicity",
-                  "widget": {
-                    "providerName": "acmeforms",
-                    "fragmentId": "field/ethnicity"
-                  }
-                },
-                {
-                  "type": "obs",
-                  "formFieldName": "obs.ACME:920",
-                  "label": "acme.app.registerPatient.language",
```

Comment:
Replace this with `"label": "Systolic blood pressure"`.

Reply:
Replaced.